tests/layout/test_layout_local_mode.py

```diff
@@ -124,7 +124,7 @@ async def test_set_local_mode(qtbot, widget_async):
     )
 
     # Sleep so the event loop can access CPU to handle the signal
-    await asyncio.sleep(5)
+    await asyncio.sleep(15)
```

Comment:
Consider making these sleep times constants.

Reply:
I am still tuning this with the updated simulation code (in `ts_m2com`). After it is in a good shape, I will unify the sleeping time in multiple constants only. Right now, it is more flexible to tune the test case based on the value directly. Thanks for this suggestion and I will do this after I am satisfied with all the M2 simulation code (`ts_m2com`). Thanks!